src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -2817,6 +2817,58 @@
                 "orchagent_zmq_port": "50"
             }
         },
+        "REMOTE_DPU": {
+            "str-8103-t1-dpu0": {
+                "type": "typeA",
+                "pa_ipv4": "192.168.2.1",
+                "pa_ipv6": "2001:db8::30",
+                "npu_ipv4": "192.168.2.10",
+                "npu_ipv6": "2001:db8::40",
+                "dpu_id": "0",
+                "swbus_port": "23606"
+            },
+            "str-8103-t1-dpu1": {
+                "type": "typeB",
+                "pa_ipv4": "192.168.2.2",
+                "pa_ipv6": "2001:db8::50",
+                "npu_ipv4": "192.168.2.20",
+                "npu_ipv6": "2001:db8::60",
+                "dpu_id": "1",
+                "swbus_port": "23607"
+            }
+        },
+        "VDPU": {
+            "vdpu0": {
+                "profile": "",
+                "tier": "",
+                "main_dpu_ids": ["dpu0"]
+            },
+            "vdpu1": {
+                "profile": "",
+                "tier": "",
+                "main_dpu_ids": ["dpu1"]
+            },
+            "vdpu2": {
+                "profile": "",
+                "tier": "",
+                "main_dpu_ids": ["dpu2"]
+            },
+            "vdpu3": {
+                "profile": "",
+                "tier": "",
+                "main_dpu_ids": ["dpu3"]
+            }
+        },
+        "DASH_HA_GLOBAL_CONFIG": {
```

Comment:
I don't think sonic recognize such config. 
Usually it should have three layer:
```
{
  "TABLE": {
    "entry": {
       "key1": "value1"
    }
  }
}
````

Reply:
I was wondering about this too. But, since there is no key specified, I wasn't sure how it should be structured.

Reply:
How about structure it like NTP?
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json#L514-L523